staging/src/k8s.io/client-go/util/workqueue/queue.go

```diff
@@ -79,7 +79,7 @@ func (s set) insert(item t) {
 	s[item] = empty{}
 }
 
-func (s set) delete(item t) {
```

Comment:
Since this is always attached to a method receiver, it never parses ambiguously.